travis/maven-settings.xml

```diff
@@ -8,6 +8,9 @@
             <id>aws.geodesy.operations</id>
```

Comment:
awsGeodesyOperations, dots are for hierarchy, not for separating words.

Reply:
Yeah, changed.